frame/babe/src/lib.rs

```diff
@@ -288,7 +294,8 @@ pub mod pallet {
 	/// execution context should always yield zero.
 	#[pallet::storage]
 	#[pallet::getter(fn lateness)]
-	pub(super) type Lateness<T: Config> = StorageValue<_, T::BlockNumber, ValueQuery>;
+	pub(super) type Lateness<T: Config> =
+		StorageValue<_, frame_system::pallet_prelude::BlockNumberFor<T>, ValueQuery>;
```

Comment:
And please import the type if you use it multiple times :see_no_evil:

Reply:
Result of find and replace 🙈. Will modify.